settings.go

```diff
@@ -24,6 +24,9 @@ type config struct {
 	AuthserviceURLPrefix *url.URL `required:"true" split_words:"true"`
 	SkipAuthURLs         []string `split_words:"true" envconfig:"SKIP_AUTH_URLS"`
 	AuthHeader           string   `split_words:"true" default:"Authorization"`
+	MultipleHosts        bool     `split_words:"true" default:"false"`
+	SchemeDefault        string   `split_words:"true" default:"https"`
+	SchemeHeader         string   `split_words:"true"`
```

Comment:
We should make this `default:"X-Forwarded-Proto"`

Reply:
Ack will add